sites/page-details/includes/footer.html

```diff
@@ -0,0 +1,34 @@
+<footer class="pagedetails{% if include.addContainer %} container{% endif %}">
+	<h2 class="wb-inv">Page details</h2>
+	{%- if page.noReportProblem -%}
+		{%- if page.share -%}
+			<div class="row">
+				<div class="wb-share col-sm-4 col-md-3 col-sm-push-8 col-md-push-9" data-wb-share='{"lnkClass": "btn btn-default btn-block"}'></div>
+				<div class="col-sm-6 col-md-5 col-lg-4 col-sm-pull-4 col-md-pull-3">
+					<dl id="wb-dtmd">
+						<dt>{{ i18nText-dateModified }}</dt>
+						<dd><time property="dateModified">{{ page.dateModified }}</time></dd>
+					</dl>
+				</div>
+			</div>
+		{%- else -%}
+			<dl id="wb-dtmd">
+				<dt>{{ i18nText-dateModified }}</dt>
+				<dd><time property="dateModified">{{ page.dateModified }}</time></dd>
+			</dl>
+		{%- endif -%}
+	{%- else -%}
+		<div class="row">
+			<div class="col-sm-6 col-md-5 col-lg-4">
```

Comment:
This might not be compatible with the future feedback. May be we can apply the limited width in the ajax vs here. And here we try to take all the available space. 

See : https://www.canada.ca/en/department-finance/economic-response-plan.html

Reply:
We used double row. One in the page-details that take the full width and another one in the ajax file that roughly represent the size of the button.

So in the future, the size of the feedback section will be determined by the row in the ajax file.